ui.apps/src/main/content/jcr_root/apps/testaem/components/customheader/customheader.html

```diff
@@ -0,0 +1,33 @@
+<!--/*
+    Copyright 2015 Adobe Systems Incorporated
+  
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+  
+        http://www.apache.org/licenses/LICENSE-2.0
+  
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/-->
+<div class="cmp-customheader" data-cmp-is="customheader" data-sly-use.customheader="com.testaem.aem.core.models.impl.CustomHeaderImpl">
```

Comment:
Because you have declared an adater interface, it's better to use the interface here, not the impl. Or else, the interface completely loses meaning. (Truth to be told, it didn't acually have much))